### PR TITLE
#1463 Macro: Some molecules are not perfect on preview tooltip

### PR DESCRIPTION
**Pull request by even1024**

### CODE REVIEW

File: core/indigo-core/molecule/monomers_basic_templates.h [hunk 1]
```diff
@@ -113,15 +113,15 @@ namespace indigo
                                          "                \"attachmentAtom\": 0,\n"
```
Comment (AliaksandrDziarkach) [inline core/indigo-core/molecule/monomers_basic_templates.h:113]: May be better use raw string for multiline strings?
Something like
char *string =  R""(multi
    "multi": "line"
 string)"";

File: core/indigo-core/molecule/src/query_molecule.cpp [hunk 2]
```diff
@@ -1029,6 +1029,11 @@ const int QueryMolecule::getTemplateAtomTemplateIndex(int idx)
     return -1;
 }
 
+void QueryMolecule::getTemplatesMap(std::unordered_map<std::pair<std::string, std::string>, std::reference_wrapper<TGroup>, pair_hash>& templates_map)
```
Comment (AliaksandrDziarkach) [inline core/indigo-core/molecule/src/query_molecule.cpp:1032]: templates_map unused

File: core/indigo-core/molecule/monomers_basic_templates.h (hunk 1 above)
Comment (even1024) [reply core/indigo-core/molecule/monomers_basic_templates.h:113]: this header will be autogenerated. here is a ticket: https://github.com/epam/Indigo/issues/1429

File: core/indigo-core/molecule/src/query_molecule.cpp (hunk 2 above)
Comment (even1024) [reply core/indigo-core/molecule/src/query_molecule.cpp:1032]: fixed

**Review by AliaksandrDziarkach — COMMENTED** [review]
LGTM

**Review by AliaksandrDziarkach — APPROVED** [review]

**Pull request merged**